documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -97,49 +97,6 @@ spec:
 .Additional resources
 For more information, refer to the xref:type-BrokerCapacity-reference[].
 
-[#auto-created-topics-logs]
-[discrete]
-== Auto-created topics and log compaction
-
-Cruise Control requires the following three topics in order to work:
-
-* `strimzi.cruisecontrol.partitionmetricsamples`
-* `strimzi.cruisecontrol.modeltrainingsamples`
-* `strimzi.cruisecontrol.metrics`
-
-These three topics are usually auto-created by Strimzi and Cruise Control. First, the metric reporters create the `strimzi.cruisecontrol.metrics` topic. 
-Then, Cruise Control creates the `strimzi.cruisecontrol.partitionmetricsamples` and `strimzi.cruisecontrol.modeltrainingsamples` topics.
-
-However, if automatic topic creation is _disabled_ in Kafka (`auto.create.topics.enable: false` is configured in `spec.kafka.config` when starting a new Kafka cluster) the Cruise Control metric reporters are unable to create the `strimzi.cruisecontrol.metrics` topic.
-In this situation, you need to manually create the `strimzi.cruisecontrol.metrics` topic before deploying Cruise Control. 
-You can do this using the following command:
-
-[source,shell,subs="attributes+"]
-----
-kubectl apply -f examples/cruise-control/cruise-control-topic.yaml
-----
-
-Log compaction might remove records that are needed by Cruise Control; therefore, all auto-created topics must be configured with `cleanup.policy=delete` to disable log compaction. 
-Cruise Control will automatically disable log compaction for the `strimzi.cruisecontrol.partitionmetricsamples` and `strimzi.cruisecontrol.modeltrainingsamples` topics.
-The Cruise Control metric reporters attempt to disable log compaction for the `strimzi.cruisecontrol.metrics` topic, but fail if they are started with a new Kafka cluster.
-This is only a problem if log compaction is enabled in Kafka with the setting `log.cleanup.policy=compact` in `spec.kafka.config`. 
-In this situation, log compaction will be enabled for the `strimzi.cruisecontrol.metrics` topic and will need to be overridden with `cleanup.policy=delete` in the `strimzi.cruisecontrol.metrics` topic.
-
-Here is an example where log compaction is disabled in a topic that was auto-created by Cruise Control:
-
-[source,yaml,subs="+quotes,attributes"]
-----
-apiVersion: kafka.strimzi.io/v1beta1
-kind: KafkaTopic
-metadata:
-  name: strimzi.cruisecontrol.metrics
-spec:
-  partitions: 1
-  replicas: 1
-  config:
-    cleanup.policy: delete
-----
-
```

Comment:
I would not remove everything. The summary and description about what topics are in place are useful I think. I would remove the parts where creating the metrics topic is mentioned.

Reply:
Agreed

Reply:
Agree that the topic description might be useful but not sure where would be the best place for it? Topic names aren't configurable AFAIK?

Reply:
Yes, the names cannot be changed but still describing what they are sounds useful to me. I would say that the current place is fine but we should make it a concept more than a reference (even if experts are @laidan6000 and @PaulRMellor here) and we should change the title. It's more about explaining why the user will see some new topics, who created them and what's their purpose.

Reply:
Agreed, @laidan6000 would it make sense to leave this removal from the configuration in this PR and for you to do a PR adding this description of the metrics topics into an appropriate section (installation/deployment)?

Reply:
Yes, good idea to keep the explanations of the CC topics as a concept. I'm happy to do this in a separate PR.